crocods-core/idsk_lite.c

```diff
@@ -129,9 +129,9 @@ char idsk_isBigEndian(void)
 #ifdef BYTE_ORDER
     return BYTE_ORDER == BIG_ENDIAN;
 #elif defined(_MSC_VER) || defined(_WIN32)
-    return 1; // It doesn't run on anything except x86, right?
+    return 1; // It doesn't run on anything except x86 and little-endian ARM, right?
```

Comment:
Counterexample: Xbox 360

Don't know if this core cares about that platform, though. If not, ignore me.

(We should add an endian detector to libretro-common. It's getting kinda tiresome seeing fifty different endian detectors, half of which are incomplete.)

Reply:
Well, in most cores I typically standardize around a single define called 'MSB_FIRST'. We could use that for big-endian detection. If it's not defined, assume little endian.

Reply:
Please don't. Currently Makefiles assume that all Linuxes are -DLSB_FIRST which is horribly wrong. We should rather not use LSB_FIRST/MSB_FIRST anymore because of their inaccuracies